README.md

```diff
@@ -0,0 +1,53 @@
+YaST SLE Migration Module
+=======================
+[![Workflow Status](https://github.com/yast/yast-migration-sle/workflows/CI/badge.svg?branch=master)](
+https://github.com/yast/yast-migration-sle/actions?query=branch%3Amaster)
+[![Jenkins Status](https://ci.opensuse.org/buildStatus/icon?job=yast-yast-migration-sle-master)](
+https://ci.opensuse.org/view/Yast/job/yast-yast-migration-sle-master/)
+[![Coverage Status](https://img.shields.io/coveralls/yast/yast-migration-sle.svg)](https://coveralls.io/r/yast/yast-migration-sle?branch=master)
+[![Inline docs](http://inch-ci.org/github/yast/yast-migration-sle.svg?branch=master)](http://inch-ci.org/github/yast/yast-migration-sle)
+
+Description
+============
+
+This YaST module allows online migration openSUSE Leap distribution to the
+SUSE Linux Enterprise Server (SLES).
+
+### Features ###
+
+- Simplified workflow for migrating openSUSE Leap installations
+  to SUSE Linux Enterprise Server (SLES).
+- Uses [yast2-registration](https://github.com/yast/yast-registration) and
+  [yast2-migration](https://github.com/yast/yast-migration) as the backend.
+
+
+Development
+===========
+
+This module is developed as part of YaST. See the
+[development documentation](http://yastgithubio.readthedocs.org/en/latest/development/).
+
+
+Getting the Sources
+===================
+
+To get the source code, clone the GitHub repository:
+
+    $ git clone https://github.com/yast/<repository>.git
+
+If you want to contribute into the project you can
+[fork](https://help.github.com/articles/fork-a-repo/) the repository and clone your fork.
+
+
+Development Environment
+=======================
+
+The module is developed only for openSUSE.

```

Comment:
only for migrating openSUSE to SLES ?

Reply:
Yes, currently openSUSE Leap -> SLES is the only supported case.